src/ui/components/Library/Team/View/Tests/hooks/useTest.ts

```diff
@@ -1,7 +1,9 @@
 import { useQuery } from "@apollo/client";
+import groupBy from "lodash/groupBy";
 import { useContext } from "react";
 
 import { GetWorkspaceTestExecutions_node_Workspace } from "shared/graphql/generated/GetWorkspaceTestExecutions";
+import { GroupedTestRun, TestExecution, TestWithExecutions } from "shared/test-suites/TestRun";
```

Comment:
```suggestion
import { GroupedTestRun } from "shared/test-suites/TestRun";
```
These two are unused.